Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp

```diff
@@ -43,9 +43,6 @@ ControllerInterface g_controller_interface;
 //
 void ControllerInterface::Initialize(void* const hwnd)
 {
-	if (m_is_init)
-		return;
```

Comment:
I have changed it to DeInit if the interface was already initialised to be safe.